CONTRIBUTING.md

```diff
@@ -187,8 +185,8 @@ please ensure that:
    which will run these checks and report any issues.
 
    If your code fails the local style checks (specifically the black
-   code formatting check) you can use `tox -eblack` to automatically
-   fix update the code formatting.
+   or rust code formatting check) you can use `tox -eblack` and
```

Comment:
I like how you corrected "rust" to "Rust" in other parts of the document, then introduced some more lower case here lol